<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lightLabel">
       <property name="text">
        <string>Light</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="color">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="intensityLabel">
           <property name="text">
            <string>Intensity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="intensity"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="removeLight">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>590</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="backgroundColor">
       <property name="text">
        <string>Background color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="noFilter">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="clipfilter">
       <property name="text">
        <string>Clip Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="shrinkfilter">
       <property name="text">
        <string>Shrink Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>590</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="objectColorLabel">
       <property name="text">
        <string>Object</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeColorItor">
       <property name="text">
        <string>red-&gt;blue-&gt;red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="objectColor">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="edgeCheck">
       <property name="text">
        <string>Edge Visibility</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="camera">
       <property name="text">
        <string>Reset camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>590</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QRadioButton" name="noShape">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="pyramid">
       <property name="text">
        <string>Pyramid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="tetrahedron">
       <property name="text">
        <string>Tetrahedron</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hexahedron">
       <property name="text">
        <string>Hexahedron</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDisplayOrientationWidget"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDisplayOrientationWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Orientation Widget</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
